content/specs/atp.md

```diff
@@ -1,294 +1,82 @@
-- [com.atproto.server](/lexicons/com-atproto-server). Account and session management.
-- [com.atproto.moderation](/lexicons/com-atproto-moderation). Moderation reporting.
-- [com.atproto.repo](/lexicons/com-atproto-repo). Repo CRUD operations.
-- [com.atproto.sync](/lexicons/com-atproto-sync). Repo content sync and streaming.
+**Identity:** account control is rooted in stable [DID](/specs/did) identifiers, which can rapidly resolved to determine the current service provider location and cryptographic keys associated with the account. [Handles](/specs/handle) provide a more human-recognizable and mutable identifier for accounts.
 
-### Additional lexicons
+**Data:** public content is stored in content-addressed and cryptographically verifiable [Repositories](/specs/repository). data records and network messages all conform to a unified [Data Model](/specs/data-model) ([IPLD](https://ipld.io/docs/data-model/), with [CBOR](https://en.wikipedia.org/wiki/CBOR) and JSON representations).
 
-For atproto to be practically useful, it needs to support a variety of sophisticated queries and behaviors. While these sophisticated behaviors could be implemented on the user device, doing so would perform more slowly than on the server. Therefore, the PDS is expected to implement lexicons which provide higher-level APIs. The reference PDS created by Bluesky implements the [bsky.app lexicon](/lexicons/bsky-app).
+**Network:** HTTP client-server and server-server [APIs](/specs/xrpc) are described with Lexicons, as are WebSocket [Event Streams](/specs/event-stream). Individual records can be referenced across the network by [AT URI](/specs/at-uri-scheme). A Personal Data Server (PDS) acts as an account's trusted agent in the network, routes client network requests, and hosts repositories. A Big Graph Server (BGS) crawls many repositories and outputs a unified event firehose.
 
-## Server-to-server API
+**Application:** APIs and record schemas for applications built on atproto are specified in [Lexicons](/specs/lexicon), which are are referenced by [Namespaced Identifiers](/specs/nsid) (NSIDs). Application-specific aggregations (such as search) are provided by an Application View (AppView) service. Clients can include mobile apps, desktop software, or web interfaces.
 
-The server-to-server APIs enable federation, event delivery, and global indexing. They may also be used to provide application behaviors such as mail delivery and form submission.
+The AT Protocol itself does not specify common social media conventions like "follows" or "avatars", leaving these to application-level Lexicons. The `com.atproto.*` Lexicons provide common APIs for things like account signup and login. These could be considered part of AT Protocol itself, though they can also be extended or replaced over time as needed. Bluesky is a Twitter-style microblogging application built on top of AT Protocol, with lexicons under the `app.bsky.*` namespace.
+
+## Cryptography
```

Comment:
kinda feel that this could warrant it's own doc

Reply:
gets a bit in the weeds for the overview docs

Reply:
@dholms: I felt like it was important to at least mention these at a high-level in the first page, to clarify the scope of these spec docs.

We will definitely need some additional docs for both the com.atproto "built-ins" and the app.bsky application stuff. They are only partially self-documenting via Lexicon.

Reply:
oh, I see you meant the Cryptography section (github PR review page wasn't clear). Yeah, cryptography could be a separate page.